library/scan_sudoers.py

```
@@ -222,6 +222,13 @@ def get_config_lines(path, params):
         sudoer_file["include_directories"] = includes["include_directories"]
     except KeyError:  # ignore keyerror
         pass
+    # process continuation lines - from the man page for sudoers:
+    #       Long lines can be continued with a backslash (‘\’) as the last
```

Ruff (0.15.13)

[warning] Comment contains ambiguous `‘` (LEFT SINGLE QUOTATION MARK). Did you mean ``` (GRAVE ACCENT)?

(RUF003)